libmscore/fret.cpp

```diff
@@ -724,22 +755,15 @@ void FretDiagram::read(XmlReader& e)
 
 //---------------------------------------------------------
 //   readNew
+//    read the new 'fretDiagram' tag
 //---------------------------------------------------------
 
 void FretDiagram::readNew(XmlReader& e)
       {
       while (e.readNextStartElement()) {
             const QStringRef& tag(e.name());
 
-            if (tag == "strings")
```

Comment:
All this stuff is handled in `read` now, because we're making sure to write properties before the new fret diagram format.